templates/kpi/deployment-worker-low-priority.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "kobo.fullname" . }}-kpi-worker-low-priority
+  labels:
+    app.kubernetes.io/component: kpi-worker-low-priority
+    {{- include "kobo.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.kpi.worker.autoscaling.enabled }}
+  replicas: {{ .Values.kpi.worker.replicaCount }}
```

Comment:
I don't think this really works. kpi/hpa-worker.yaml is what scales the "regular" worker. But it won't apply here. So if autoscaling is enabled, replicas just isn't set at all.

We probably should duplicate low priority worker settings entirely. I don't love having yet more settings.